funcbench/env.go

```diff
@@ -0,0 +1,229 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/github"
```

Comment:
We will use `github.com/google/go-github/v29/github`.

Reply:
When it was decided and why? (:

Reply:
@nevill let's keep those changes for another pr like I mentioned in my previous comment. This pr is already is adding a lot of things.

Reply:
@bwplotka one of the tools (commentMonitor) was using the `repository_dispatch` event, v29 adds official support for it in the api so we would not want to use multiple versions of go-github here. So it will be just a refactoring change, would not really make it part of this pr if not really required (previous comment)

Reply:
I see, if we could do this in separate PR it would be awesome :heart: